modules/oe_theme_content_event/src/Plugin/PageHeaderMetadata/EventContentType.php

```diff
@@ -32,15 +75,15 @@ public function applies(): bool {
   public function getMetadata(): array {
     $metadata = parent::getMetadata();
 
-    // Use business logic from "list_default" formatter.
     $node = $this->getNode();
-    $provider = $node->get('oe_event_type')->getFieldDefinition()
-      ->getFieldStorageDefinition()
-      ->getOptionsProvider('value', $node);
+    $metadata['metas'] = [];
```

Comment:
Let's remove this, it's not needed. Also isn't oe_event_type required? We weren't checking for it's presence before:
```php
    $metadata['metas'] = [
      $provider->getPossibleOptions()[$node->get('oe_event_type')->value],
    ];
```

Reply:
Yes, but if we create an event with api, like behat without type it will fail.